main/package.mill

```diff
@@ -100,7 +100,13 @@ object `package` extends RootModule with build.MillStableScalaModule with BuildI
 
   object util extends build.MillStableScalaModule {
     def moduleDeps = Seq(api, client)
-    def ivyDeps = Agg(build.Deps.coursier, build.Deps.jline)
+    def ivyDeps = Agg(
+      build.Deps.coursier,
+      build.Deps.coursierJvm.exclude(
+        "com.github.alexarchambault" -> "argonaut-shapeless_6.3_2.13",
```

Comment:
Please add a comment here explaining why it is excluded, for future reference

Reply:
Note that bumping coursier to `2.1.15` or higher should make that exclude unnecessary (trying that in https://github.com/com-lihaoyi/mill/pull/3943)

Reply:
Seems you did it already in this PR, so the exclude can be removed